apps/api/src/app/notification-template/notification-template.controller.ts

```diff
@@ -59,6 +59,7 @@ export class NotificationTemplateController {
   @ApiOperation({
     summary: 'Get workflows',
```

Comment:
```suggestion
    summary: 'Get notification templates',
```
Should we change this description back to notification templates to show it in the swagger deprecated section?